icons/outline/align-left-2.svg

```diff
@@ -0,0 +1,16 @@
+<svg
```

Comment:
You can add a comment on top with tabs and category to make the icon easier to find:, for example:

```svg
<!--
tags: [flat, office, city, urban, scyscraper, architecture, construction]
category: Buildings
-->
<svg
```